hack/lib/vars.bash

```diff
@@ -29,10 +29,11 @@ PREVIOUS_CSV="$(metadata.get project.name).v$PREVIOUS_VERSION"
 export CURRENT_VERSION PREVIOUS_VERSION CURRENT_CSV PREVIOUS_CSV
 
 # Directories below are filled with source code by ci-operator
-export KNATIVE_SERVING_HOME="${GOPATH}/src/knative.dev/serving"
-export KNATIVE_EVENTING_HOME="${GOPATH}/src/knative.dev/eventing"
-export KNATIVE_EVENTING_KAFKA_BROKER_HOME="${GOPATH}/src/knative.dev/eventing-kafka-broker"
-export KNATIVE_EVENTING_ISTIO_HOME="${GOPATH}/src/knative.dev/eventing-istio"
+export TEST_SOURCE_BASE_PATH=${TEST_SOURCE_BASE_PATH:-"/go/src/knative.dev"}
```

Comment:
Giving the option to override the base path just in case we're relying on that in downstream testing